wazo_sysconfd/helpers.py

```diff
@@ -33,7 +32,7 @@ def extract_scalar_from_list(xlist):
 
 def extract_scalar_from_dict(xdict):
     """ Extract scalar values from a dict natural ordered by key """
-    return [xdict[key] for key in sorted(xdict.iterkeys(), natsort)
+    return [xdict[key] for key in sorted(iter(xdict.keys()), natsort)
```

Comment:
No need for `iter` here, is there?

Reply:
indeed, that was an automatic transformation done py 2to3 :smile: I will fix it